gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-client/src/test/java/org/apache/gobblin/service/FlowConfigV2Test.java

```diff
@@ -160,6 +160,8 @@ public void testCheckFlowExecutionId() throws Exception {
   @Test
   public void testPartialUpdate() throws Exception {
     FlowId flowId = new FlowId().setFlowGroup(TEST_GROUP_NAME).setFlowName(TEST_FLOW_NAME_3);
+    ServiceRequester testRequester = new ServiceRequester("testName", "USER_PRINCIPAL", "testFrom");
```

Comment:
nitpick but do we want to add these as constants if they are reused?

Reply:
Good point, updated the tests that don't need to modify the requester to use a constant